providers/os/resources/packages/windows_packages.go

```diff
@@ -268,6 +283,30 @@ func (w *WinPkgManager) getInstalledApps() ([]Package, error) {
 	return ParseWindowsAppPackages(cmd.Stdout)
 }
 
+func (w *WinPkgManager) getAppxPackages() ([]Package, error) {
+	canRunCmd := w.conn.Capabilities().Has(shared.Capability_RunCommand)
+	// we always prefer to use the powershell command to get the appx packages, fallback to filesystem if not possible
+	if !canRunCmd && (w.conn.Type() == shared.Type_FileSystem || w.conn.Type() == shared.Type_Device) {
+		return w.getFsAppxPackages()
+	}
+
+	b, err := windows.Version(w.platform.Version)
```

Comment:
I think we want to create a seperate function like `getPwshAppxPackages()`. Then the function is easier to read.

Reply:
good point, done